openapi/integrator/merchant/partner.yaml

```diff
@@ -273,3 +287,9 @@ components:
         - Approved
         - Declined
         - Failed
+
+    stan:
+      type: string
+      maxLength: 6
+      description: >-
+        System Trace Audit Number (STAN): This field is a number assigned by the message originator to identify the transaction.
```

Comment:
`System trace audit number; a number assigned by the payment initiator to identify the transaction.`

Reply:
Sure. I found this in the NISOK description of the field.